mirage/views/resource.ts

```diff
@@ -1,10 +1,30 @@
 import { HandlerContext, Request, Response, Schema } from 'ember-cli-mirage';
 
+export function createResource(this: HandlerContext, schema: Schema) {
+    const attrs = {
+        ...this.normalizedRequestAttrs('resource'),
+        finalized: false,
+    };
+    return schema.resources.create(attrs);
+}
+
 export function updateResource(this: HandlerContext, schema: Schema, request: Request) {
     const resource = schema.resources.find(request.params.id);
     const { data: { attributes } } = JSON.parse(request.requestBody);
```

Comment:
Should we do `const attrs = this.normalizedRequestAttrs('request')` to camelize the attr keys?

Reply:
Maybe? Lemme try that. Perhaps we can eliminate one of the if blocks.

Reply:
🤞 I think if nothing weird happens with the attr names, we can just check the pid's formate, and then do `resource.update({attrs});`, I think?